chart/portworx/templates/_helpers.tpl

```diff
@@ -221,3 +221,8 @@ Populate the ports based on deployemnt environment
     {{- printf "9021" -}}
 {{- end -}}
 {{- end -}}
+
+{{- define "px.clusterName" -}}
+{{ $match := regexReplaceAll "[^A-Za-z0-9._-]+" .Values.clusterName "" }}
```

Comment:
clusterName cannot be nil, return an error.

Reply:
resolved